ppl/cmd/zar/README.md

```diff
@@ -101,10 +99,10 @@ zar map "count()" | zq -f table -
 which results in:
 ```
 COUNT
-188697
-436829
-409224
-427328
+275745
+276326
+290789
+619218
```

Comment:
Here's the change in record distribution across chunks as mentioned in the previous comment.